### PR TITLE
docs: add gitbar app

### PR DESCRIPTION
**Pull request by mikaelkristiansson**
This is the link to the project: [GitBar](https://github.com/mikaelkristiansson/gitbar)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [x] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

**Comment by mikaelkristiansson** [comment]
This is my first Tauri app, and I really loved creating it! ❤️ 
Hopefully it's a useful app to someone else than just me.

**Comment by JonasKruckenberg** [comment]
Just one point of feedback regarding your app:
The allowlist option:
```json
"all": true,
```
Overrides all other options, so your config still enables all APIs

File: README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Link Saas](https://github.com/linksaas/desktop) - Efficiency tools for software development teams.
 - [Dropcode](https://github.com/egoist/dropcode) - Simple and lightweight code snippet manager.
 - [JournalV](https://github.com/ahmedkapro/journalv) - Journaling app for your days and dreams.
+- [GitBar](https://github.com/mikaelkristiansson/gitbar) - Tray bar app for github reviews
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

**Comment by mikaelkristiansson** [comment]
> Just one point of feedback regarding your app: The allowlist option:
> 
> ```json
> "all": true,
> ```
> 
> Overrides all other options, so your config still enables all APIs

Great feedback, I’ll change it 👍

**Comment by mikaelkristiansson** [comment]
> Just one point of feedback regarding your app: The allowlist option:
> 
> ```json
> "all": true,
> ```
> 
> Overrides all other options, so your config still enables all APIs

https://github.com/mikaelkristiansson/gitbar/pull/9

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
Alright, thanks for being part of the community 😁

**Pull request merged**